src/index.html

```diff
@@ -1,13 +1,450 @@
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="images/title-wave.svg">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="header__top">
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.svg"
+                alt="bose"
+              >
+            </a>
+            <div class="header__nav-list">
+              <a
+                href="tel:+1 234 555-55-55"
+                class="icon icon--call"
+              >
+                <div class="icon__tooltip">
+                  +1 234 555-55-55
+                </div>
+              </a>
+
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+
+          <h1 class="header__title">
+            The world shades.
+            <br>
+            Your music shines.
+          </h1>
+        </div>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="menu__content">
+          <div class="menu__top">
+            <a href="#" class="logo">
+              <img
+                class="logo__image"
+                src="images/logo.svg"
+                alt="bose"
+              >
+            </a>
+            <a href="#" class="icon icon--close"></a>
+          </div>
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#about-us" class="menu__link">About Us</a>
+            </li>
+            <li class="menu__item">
+              <a href="#now-on-view" class="menu__link">Now on view</a>
+            </li>
+            <li class="menu__item">
+              <a href="#testimonials" class="menu__link">testimonials</a>
+            </li>
+            <li class="menu__item">
+              <a href="#Contacts" class="menu__link">Contacts</a>
+            </li>
+          </ul>
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--phone"
+          >
+            +1 234 5555-55-55
+          </a>
+          <a
+            href="tel:+1 234 5555-55-55"
+            class="menu__call menu__call--action"
+          >
+            call to order
+          </a>
+        </div>
+      </div>
+    </nav>
+
+    <main>
+      <section class="page__section recommends" id="about-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Recommended
+          </h2>
+
+          <div class="grid grid--tablet">
+            <div class="recommends__item grid__item grid__item--tablet-2-5 grid__item--desktop-1-4">
```

Comment:
this block is element from `grid` block, not `recommends`. So `grid__item` is good class
```suggestion
            <div class="grid__item grid__item--tablet-2-5 grid__item--desktop-1-4">
```

Reply:
Здесь нужен именно  recommends__item потому что я задаю ним нижние марджини исключительно для трех подразделов секции рекомендс, а grid__item я испольщую очень много где на странице а не только в разделе рекомендс